### ISSUE TITLE
Trying to Connect to an Closed Channel

### ISSUE THREAD

**Issue by KAGamerAlex** [rmichela/Bukkit-SSHD#10]
Hey there, I noticed you tried to code around this but it looks like this error still appears in console.  I'm running the latest release available from Spigot, 1.3.4.  This tends to happen when I use WinSCP for SFTP.  I'm guessing this is because it opens a new SSH connection to the server to transfer a file.  Any tips?
```
16.04 20:15:32 [Server] ERROR Error sending message to SSHDCommandSender
16.04 20:15:32 [Server] INFO org.apache.sshd.common.SshException: write(ChannelOutputStream[ChannelSession[id=0, recipient=256]-ServerSessionImpl[KAGamer@/{redactedIP}]] SSH_MSG_CHANNEL_DATA) len=2 - channel already closed
16.04 20:15:32 [Server] INFO at org.apache.sshd.common.channel.ChannelOutputStream.write(ChannelOutputStream.java:105) ~[SSHD-1.3.4.jar:?]
16.04 20:15:32 [Server] INFO at com.ryanmichela.sshd.FlushyOutputStream.write(FlushyOutputStream.java:34) ~[SSHD-1.3.4.jar:?]
16.04 20:15:32 [Server] INFO at sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:221) ~[?:1.8.0_151]
16.04 20:15:32 [Server] INFO at sun.nio.cs.StreamEncoder.implWrite(StreamEncoder.java:282) ~[?:1.8.0_151]
16.04 20:15:32 [Server] INFO at sun.nio.cs.StreamEncoder.write(StreamEncoder.java:125) ~[?:1.8.0_151]
16.04 20:15:32 [Server] INFO at java.io.OutputStreamWriter.write(OutputStreamWriter.java:207) ~[?:1.8.0_151]
16.04 20:15:32 [Server] INFO at java.io.Writer.write(Writer.java:127) ~[?:1.8.0_151]
16.04 20:15:32 [Server] INFO at jline.console.ConsoleReader.print(ConsoleReader.java:3440) ~[SSHD-1.3.4.jar:?]
16.04 20:15:32 [Server] INFO at jline.console.ConsoleReader.println(ConsoleReader.java:3462) ~[SSHD-1.3.4.jar:?]
16.04 20:15:32 [Server] INFO at com.ryanmichela.sshd.implementations.SSHDCommandSender.sendRawMessage(SSHDCommandSender.java:36) [SSHD-1.3.4.jar:?]
16.04 20:15:32 [Server] INFO at com.ryanmichela.sshd.implementations.SSHDCommandSender.sendMessage(SSHDCommandSender.java:30) [SSHD-1.3.4.jar:?]
16.04 20:15:32 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.CraftServer.broadcast(CraftServer.java:1286) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.CraftServer.broadcastMessage(CraftServer.java:458) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at org.bukkit.Bukkit.broadcastMessage(Bukkit.java:281) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at me.minebuilders.clearlag.Util.broadcast(Util.java:35) [Clearlag.jar:?]
16.04 20:15:32 [Server] INFO at me.minebuilders.clearlag.tasks.ClearTask.run(ClearTask.java:44) [Clearlag.jar:?]
16.04 20:15:32 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:739) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot131.jar:git-Spigot-7754231-94b0980]
16.04 20:15:32 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_151]
```

**Comment by Haarolean** [comment]
Yep, turns out still not fixed. We'll take a look into this again.

**Comment by Haarolean** [comment]
Fixed in 1.3.4.1.
[Download v1.3.4.1](https://github.com/rmichela/Bukkit-SSHD/releases/tag/1.3.4.1)
